### PR TITLE
Improve tag handling

### PR DESCRIPTION
**Pull request by arapulido**
Improve DD_TAGS handling by the buildpack:

* Allow for space separated tags
* Avoid removing DD_TAGS content
* Allow for DD_TAGS to be extended in the prerun.sh script

Fixes: #160 #102

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ In addition to the environment variables shown above, there are a number of othe
...
-| `DD_TAGS`                  | *Optional.* Sets additional tags provided as a comma-delimited string. For example, `heroku config:set DD_TAGS="simple-tag-0, tag-key-1:tag-value-1"`. The buildpack automatically adds the tags `dyno` which represent the dyno name (e.g. web.1) and `dynotype` (the type of dyno, e.g `run` or `web`). See the ["Guide to tagging"][8] for more information.                                                                                                                                                             |
+| `DD_TAGS`                  | *Optional.* Sets additional tags provided as a space-separated string (**Note**: comma-separated string for buildpack versions `1.16` and earlier). For example, `heroku config:set DD_TAGS="simple-tag-0 tag-key-1:tag-value-1"`. The buildpack automatically adds the tags `dyno` which represent the dyno name (e.g. web.1) and `dynotype` (the type of dyno, e.g `run` or `web`). See the ["Guide to tagging"][8] for more information.                                                                                                                                                             |
```
Comment (burningion) [inline README.md:82]: What's the rationale behind the switch from comma separated strings to space separated strings? 

And looking at the tests, it appears as if both are supported?

Comment (arapulido) [reply README.md:82]: https://github.com/DataDog/heroku-buildpack-datadog/issues/160#issuecomment-590518035

**Comment by arapulido** [comment]
@burningion any other comments? Thanks!

**Comment by burningion** [comment]
Nope looks good, thanks for the clarification!

**Pull request merged**